<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cab_AddDlg</class>
 <widget class="QDialog" name="Cab_AddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>228</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新增机柜</string>
  </property>
  <widget class="QWidget" name="widget_white" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>176</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_describe">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>请填写新增机柜详细信息：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>45</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>机柜名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cabEdit">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>45</y>
      <width>135</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>40</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>高度：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSpin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>60</width>
      <height>33</height>
     </rect>
    </property>
    <property name="suffix">
     <string>U</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="value">
     <number>42</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>85</y>
      <width>126</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>主要PDU IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit_1">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>85</y>
      <width>135</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>85</y>
      <width>106</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string> 设备类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCb_1">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>85</y>
      <width>98</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IP-PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SI-PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MPDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RPDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NPM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ZPDU</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_deputy">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>副机：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>125</y>
      <width>126</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>备用PDU IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipEdit_2">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>125</y>
      <width>135</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>125</y>
      <width>106</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string> 设备类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCb_2">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>125</y>
      <width>98</width>
      <height>33</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IP-PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SI-PDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MPDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RPDU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NPM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ZPDU</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_deputy_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>副机：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="powSpin">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="suffix">
     <string>kW</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="value">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>90</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>额定功率：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>主机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>副机</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>87</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>主机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>副机</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 199, 249);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 199, 249);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deputyDescribe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>副机（0代表：主机，1-9代表：副机序号）</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cabEdit</tabstop>
  <tabstop>heightSpin</tabstop>
  <tabstop>ipEdit_1</tabstop>
  <tabstop>ipEdit_2</tabstop>
  <tabstop>okBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
